packages/dds/tree/src/core/rebase/changeRebaser.ts

```diff
@@ -51,6 +51,9 @@ export interface ChangeRebaser<TChangeset> {
 	compose(changes: TaggedChange<TChangeset>[]): TChangeset;
 
 	/**
+	 * @param changes - The changes to invert.
+	 * @param isRollback - Whether the inverted change is meant to rollback a change on a branch as is the case when
+	 * performing a sandwich rebase.
```

Comment:
Might be worth adding a sentence about why this is relevant or how this flag affects the behavior.